client/components/feature-flag/helpers.spec.js

```diff
@@ -0,0 +1,36 @@
+import { isEnabled, mapFlagsToHash } from './helpers';
+
+describe('feature-flag helpers', () => {
+  describe('isEnabled', () => {
+    it('When passed name = "workflow-terminate" and flagHash = {} it should return false.', () => {
```

Comment:
Nit: Reformat to read like a sentence and maybe generalize, ex. "returns false when field does not exist in flagHash"

Reply:
updated